sci-physics/jaxodraw-bin/jaxodraw-bin-2.1.0.ebuild

```diff
@@ -0,0 +1,71 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+JAVA_PKG_IUSE="doc source"
+
+inherit java-pkg-2 java-ant-2 desktop
+
+MY_PN="jaxodraw"
+MY_PV=$(ver_rs 2 '-')
+MY_P=${MY_PN}-${MY_PV}
+
+DESCRIPTION="Java program for drawing Feynman diagrams"
+HOMEPAGE="
+	https://jaxodraw.sourceforge.io/index.html
+	http://jaxodraw.sourceforge.net/
+"
+SRC_URI="
+	https://${MY_PN}.sourceforge.net/download/pkgs/${MY_P}-bin.tar.gz
+	https://${MY_PN}.sourceforge.net/download/pkgs/${MY_P}-src.tar.gz
+	latex? ( https://downloads.sourceforge.net/${MY_PN}/axodraw4j_2008_11_19.tar.gz )
+"
+S="${WORKDIR}/${MY_P}"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+latex"
+
+DEPEND="
+	>=virtual/jdk-1.6
```

Comment:
We don't have 1.6 in the tree anymore so we can't test with that version, let's make it 1.8 instead.